docs/source/upgrade_notes.rst

```diff
@@ -677,302 +677,3 @@ Upgrade Notes
 
 * ``st2ctl reload`` now also registers rules by default. Prior to this release actions, aliases,
   sensors, triggers and configs were registered. Now rules are also registered by default.
-
-|st2| v1.6
```

Comment:
All good except of removing history behind the `st2` `v2.0`. That's still "history".

Any particular thinking behind cleaning that out or we can keep it for future?

Reply:
Part of the problem was I was clearing out the "all_in_one" doc. 

Rather than figuring out how to update all of the update notes, it seemed easier to just remove upgrade notes, since 1.6 is ~3 years old now. 

People can still go and access old versions of docs if they really need.

Reply:
Sounds reasonable :+1: